### ISSUE TITLE
想提醒一个事实：用内部的codebase训练，然后自己随便改一下根本无法复现的代码很浪费社区的时间

### ISSUE THREAD

**Issue by luohao123** [LingyvKong/OneChart#4]
且涉嫌学术造假。

从Vary 到这个代码，我想问你们真的跑过么？没有一个能复现的，根本训练无法收敛。

你们可以不开源数据，但起码需要又一点对学术的严谨和负责，开放个代码但是没法训练，请问开了个寂寞？让人不得不怀疑这当中的gap，以及动机。

<img width="596" alt="image" src="https://github.com/LingyvKong/OneChart/assets/49749220/a418cefd-4e6c-44ca-881f-4172c77faef1">

这非常ying影响社区对这一系列工作的认可度，以及对国人开源行为的质疑。

**可以不开源，但是没有必要开源一个不收敛的代码，然后说自己的内部的codebase可以训，除非你们开源的base能对齐，否则大家跟这一些工作意义是什么？**

<img width="552" alt="image" src="https://github.com/LingyvKong/OneChart/assets/49749220/305cd551-5e3c-4b1c-a1d1-82dbac505229">

作为一个学者，必要严谨性要有，纵观你们整个开源的工作，在README里面没有任何一点训练的细节，如何从零复现？如果只是开源一个权重，这能叫开源吗？这工作的意义又在哪里呢？

如果不想开源，能不能不要开园一个无法训练的代码？**这非常浪费社区大家宝贵的时间，都是中国人，没有必要这么内耗**

**Comment by LingyvKong** [comment]
1. The amount of training data, batch size, and learning rate have been clearly stated in the OneChart paper.
2. Compared to the code I used, this code has almost only changed certain variable names, debug histories and hard paths for clear. Maybe there are still some bugs, but I have no reason to intentionally do so and will continue to correct them, just like most open source code. Please do not be preconceived and maliciously speculate about others.
3. Report bugs. Please provide a detailed explanation of your situation, such as your environment, data size, learning rate, etc. Clear bugs will be prioritized for resolution. Some bugs have been resolved with the help of the open source community, please try it yourself.
4. Keep polite, I will do my best to solve all problems in my spare time.

**Comment by Zxr1314** [comment]
能开源就不错了，都是吃力不讨好的事情，伸手党怎么那么多事儿呢。
